tests/staking/observable/createStakingTx.test.ts

```diff
@@ -0,0 +1,164 @@
+import { ObservableStaking } from "../../../src";
+import { testingNetworks } from "../../helper";
+import { ObservableStakingParams } from "../../../src/types/params";
+import { UTXO } from "../../../src/types/UTXO";
+import { StakingError, StakingErrorCode } from "../../../src/error";
+import { BTC_DUST_SAT } from "../../../src/constants/dustSat";
+import { RBF_SEQUENCE } from "../../../src/constants/psbt";
+import * as stakingUtils from "../../../src/utils/staking";
+import * as stakingScript from "../../../src/staking/stakingScript";
+import * as staking from "../../../src/staking";
+
+describe("Create staking transaction", () => {
+  const { network, networkName, dataGenerator } = testingNetworks[0];
+
+  let stakerInfo: { address: string, publicKeyNoCoordHex: string, publicKeyWithCoord: string };
+  let finalityProviderPublicKey: string;
+  let params: ObservableStakingParams;
+  let stakingTerm: number;
+  let utxos: UTXO[];
+  const feeRate = 1;
```

Comment:
Could this be a random fee rate?

Reply:
If they're already tested, you are not performing the same checks. You are just assigning a random fee rate and assume that it works.